public/apps/account/tenant-switch-panel.tsx

```diff
@@ -297,8 +298,18 @@ export function TenantSwitchPanel(props: TenantSwitchPanelProps) {
 
           <EuiButton
             data-test-subj="confirm"
-            fill
-            disabled={!isMultiTenancyEnabled}
+            fill={
```

Comment:
Can we move the logic into a local variable it is not duplicated it on both lines?

Reply:
Thanks for the review! I just made an update on this.